apps/condo/package.json

```diff
@@ -121,12 +118,12 @@
     "rc-picker": "^4.2.0",
     "rc-table": "~7.25.3",
     "rc-upload": "~4.3.0",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
+    "react": "^17.0.2",
```

Comment:
Why not 18? Is there nextjs blocker?

Reply:
Next 13+ requires React 18+.
React 18+ requires giant codemod (basically you need form 18.x specify children in props explicitly)

So, 17 is our target for now

Reply:
That's acceptable